sites/en/intro-to-rails/running_your_application_locally.step

```diff
@@ -6,18 +6,23 @@ end
 steps do
   step do
     console "rails server"
+    message "This will print some stuff and stay running forever, printing more stuff
+every time you visit a page in your app."
   end
   step do
     text "Point your web browser to "
     url "http://localhost:3000" 
-    p "See your web app actually running!"
+    message "See your web app actually running!"
```

Comment:
the `p` vs `message` really only matters if there's markdown in the string, but whatevs.